cg/cli/upload/nipt/base.py

```diff
@@ -1,19 +1,86 @@
 """ Upload NIPT results via the CLI"""
 
 import logging
+import sys
+import traceback
+from datetime import datetime
+from typing import Optional
 
-from .ftp import ftp
-from .statina import statina
+from cg.cli.upload.utils import suggest_cases_to_upload
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from cgmodels.cg.constants import Pipeline
+
+from .ftp import ftp, nipt_upload_case
+from .statina import statina, batch
 
 import click
 
 LOG = logging.getLogger(__name__)
 
 
-@click.group()
-def nipt():
+@click.group(invoke_without_command=True)
+@click.option("-c", "--case", "case_id", help="Upload to all apps")
+@click.pass_context
+def nipt(context: click.Context, case_id: Optional[str]):
     """Upload NIPT result files"""
-    pass
+
+    LOG.info("*** NIPT UPLOAD START ***")
+
+    if context.invoked_subcommand is not None:
+        return
+
+    config_object: CGConfig = context.obj
+    status_db: Store = config_object.status_db
+
+    if not case_id:
+        suggest_cases_to_upload(status_db=status_db, pipeline=Pipeline.FLUFFY)
+        raise click.Abort
```

Comment:
I think this functionality should be covered by uploading in dry-run. Uploading in dry-run should tell which cases would be uploaded (same as suggesting).

Reply:
fixed